mobile/lib/ui/viewer/file/video_control.dart

```diff
@@ -1,454 +1,449 @@
-          border: Border.all(
-            color: strokeFaintDark,
-            width: 1,
-          ),
-        ),
-        child: GestureDetector(
-          behavior: HitTestBehavior.opaque,
-          onTap: onPressed,
-          child: AnimatedSwitcher(
-            duration: const Duration(milliseconds: 250),
-            transitionBuilder: (Widget child, Animation<double> animation) {
-              return ScaleTransition(scale: animation, child: child);
-            },
-            switchInCurve: Curves.easeInOutQuart,
-            switchOutCurve: Curves.easeInOutQuart,
-            child: isPlaying
-                ? const Icon(
-                    Icons.pause,
-                    size: 32,
-                    key: ValueKey("pause"),
-                    color: Colors.white,
-                  )
-                : const Icon(
-                    Icons.play_arrow,
-                    size: 36,
-                    key: ValueKey("play"),
-                    color: Colors.white,
-                  ),
-          ),
-        ),
-      ),
-    );
-  }
-}
+// // ignore_for_file: implementation_imports
+
+// import 'dart:async';
+
+// import "package:chewie/chewie.dart";
+// import "package:chewie/src/notifiers/index.dart";
+// import 'package:flutter/material.dart';
+// import "package:photos/models/file/file.dart";
+// import "package:photos/theme/colors.dart";
+// import "package:photos/theme/ente_theme.dart";
+// import "package:photos/ui/common/loading_widget.dart";
+// import "package:photos/ui/viewer/file/preview_status_widget.dart";
+// import "package:photos/ui/viewer/file/video_control/custom_progress_bar.dart";
+// import "package:photos/utils/seconds_to_duration.dart";
+// import 'package:provider/provider.dart';
+// import 'package:video_player/video_player.dart';
+
+// class VideoControls extends StatefulWidget {
+//   const VideoControls({
+//     super.key,
+//     required this.file,
+//     required this.onStreamChange,
+//     required this.playbackCallback,
+//   });
+//   final EnteFile file;
+//   final void Function()? onStreamChange;
+//   final void Function(bool)? playbackCallback;
+
+//   @override
+//   State<StatefulWidget> createState() {
+//     return _VideoControlsState();
+//   }
+// }
+
+// class _VideoControlsState extends State<VideoControls>
+//     with SingleTickerProviderStateMixin {
+//   late PlayerNotifier notifier;
+//   late VideoPlayerValue _latestValue;
+//   Timer? _hideTimer;
+//   Timer? _initTimer;
+//   Timer? _showAfterExpandCollapseTimer;
+//   bool _dragging = false;
+//   bool _displayTapped = false;
+//   Timer? _bufferingDisplayTimer;
+//   bool _displayBufferingIndicator = false;
+
+//   final barHeight = 48.0 * 1.5;
+//   final marginSize = 5.0;
+
+//   late VideoPlayerController controller;
+//   ChewieController? _chewieController;
+
+//   // We know that _chewieController is set in didChangeDependencies
+//   ChewieController get chewieController => _chewieController!;
```

Comment:
Can delete files instead of commenting it out? Can always pull it from git commit history.

Reply:
Done!